backend/api/geocoder.Development.json

```diff
@@ -1,5 +1,5 @@
 {
   "Geocoder": {
-    "HostUri": "https://apps.gov.bc.ca/pub/geocoder"
+    "HostUri": "https://geocoder.api.gov.bc.ca"
```

Comment:
This URL requires a developer key.  If you want to reuse mine, just let me know and I'll pass it along.